src/plugins/intel_cpu/src/config.h

```diff
@@ -26,6 +26,7 @@ struct Config {
     bool enableDynamicBatch = false;
     std::string dumpToDot = "";
     int batchLimit = 0;
+    size_t rtCacheCapacity = 100ul;
```

Comment:
We should consider two scenarios here. The first one is a cache hit, and the cost will be O(1) in the both cases since we search through a hash table and reorder one element in a double-linked list. In case of cache miss, there will be difference. In case when we have only one element, we just add another with constant complexity. But in the case of full cache, to the convectional action of a new element inserting we have to erase one element from the underlying hash table and one entry from the double-linked list, which has O(1) complexity also, but it is an extra action.